apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -38,14 +39,14 @@
 
 patterns() ->
     [{?MODULE, F, A, []} || {F, A} <- [ {handle_request, 3}
-				      , {forbidden, 1}
+				      , {forbidden, 2}
 				      , {ok_response, 1}
 				      , {process_request, 2}
 				      , {dry_run_results, 1}]].
 
--spec forbidden( OperationID :: atom() ) -> boolean().
-forbidden(OpId) ->
-    OpSpec = endpoints:operation(OpId),
+-spec forbidden( Mod :: module(), OperationID :: atom() ) -> boolean().
+forbidden(Mod, OpId) ->
+    OpSpec = Mod:operation(OpId),
```

Comment:
Unrelated bug: we were able to block API groups (like `debug`) only if those `OperationIDs` were present in `swagger.yaml`, regardless if the call comes from `/v2/` or `/v3/`. Now `oas3.yaml` is being used for `/v3/` and `swagger.yaml` for `/v2/` as it should